kavm/src/kavm/pyk_utils.py

```diff
@@ -36,12 +36,30 @@ def tvalue(value: Union[str, int, bytes]) -> KInner:
     else:
         raise TypeError()
 
+def tvalue_elem(value: Union[str, int, bytes]) -> KInner:
+    if type(value) is int:
+        return intToken(value)
+    elif type(value) is OnComplete:
+        return intToken(int(value))
+    elif type(value) is str:
+        return stringToken(value)
+    elif type(value) is bytes:
+        return stringToken(value.decode())
+    else:
+        raise TypeError()
+
 
 def tvalue_list(value: List[Union[str, int, bytes]]) -> KInner:
     if len(value) == 0:
         return KApply('.TValueList')
-    else:
-        raise NotImplementedError()
+    if len(value) == 1:
+        return tvalue_elem(value[0])
+    return KApply('#abcd', [tvalue_elem(value[0]), tvalue_list(value[1:])])
+#      return build_cons(
+#                 unit=KApply('.TValueList'),
+#                 label=KLabel('#abcd'),
+#                 terms=[tvalue(v) for v in value]
+#             )
```

Comment:
It would be cleaner if I can figure out how to just use `build_cons` here to do the same thing